sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+### Features Added
+
+Initial release
```

Comment:
Consistency with other Changelogs: the entries should be a bulleted list with complete sentences.

```suggestion
- Initial release.
```